docs/source/introduction/quickstart.ipynb

```diff
@@ -22,45 +22,56 @@
     "\n",
```

Comment:
<p>I am not sure I understand the point about the online algorithm. It always comes down to limited computational resources in whatever measure (wallclock time, cpu cycles, ...). I find the formulation with "fed" a little strange. </p><p> </p><p>If you want to give examples, I think it is convincing enough to say something along the lines of: "Many applications in practice use linear systems which are so large, that running to convergence simply takes way too long even on large-scale hardware."</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>I meant real-time systems that run on a clock and cannot wait until a numerical procedure converged, but it;s not so important. I just used your sentence. </p>